src/pal/tests/palsuite/runpaltests.sh

```diff
@@ -38,16 +38,35 @@ PAL_TEST_LIST=$PWD/paltestlist.txt
 cd $OLDPWD
 echo The list of PAL tests to run will be read from $PAL_TEST_LIST
 
-# Create path to a folder where PAL tests will store their output
-# First check if the output folder was passed as a parameter to the script.
-# It is supposed be the second parameter so check if we have more than 1 argument.
+# Create the test output root directory
+mkdir -p /tmp/PalTestOutput
```

Comment:
Won't this block cause problems for multiple jobs running on the same machine at the same time?